.circleci/config.yml

```
@@ -26,12 +26,9 @@ defaults: &defaults
   working_directory: ~/repo
 
 jobs:
-  build_elixir_1_11_otp_23:
+  build_elixir_1_12_otp_24:
     docker:
-      - image: erlang:23.1
-        environment:
-          ELIXIR_VERSION: 1.11.0-otp-23
-          LC_ALL: C.UTF-8
+      - image: hexpm/elixir:1.12.0-rc.1-erlang-24.0-alpine-3.13.3
```

```suggestion
docker:
      - image: erlang:24.0
        environment:
          ELIXIR_VERSION: 1.12.0-rc.1-otp-23
          LC_ALL: C.UTF-8
```

This project isn't using the hexpm images, so more changes would be needed. I think what I posted above should work.